bugbug/db.py

```diff
@@ -41,10 +42,17 @@ def is_old_version(path):
     return DATABASES[path]["version"] > prev_version
 
 
-def extract_file(path):
+def extract_file(path, compression_type):
     with open(path, "wb") as output_f:
```

Comment:
I'd do: compression_type = os.path.splitext(path)[1], so you don't need to pass it as a parameter.